bin/git-info

```diff
@@ -1,47 +1,64 @@
 #!/usr/bin/env bash
 
-get_config() {
-  git config --list
-}
+clear
 
-most_recent_commit() {
-  git log --max-count=1 --pretty=short
-}
 
-local_branches() {
-  git branch
-}
+max_count=1
+if [[ "$1" == "-c" ]]; then
+    shift
+    max_count="$1"; shift
+fi
 
-remote_branches() {
-  git branch -r
-}
+color="--color=always"
+no_color="--no-color"
+if [[ "$1" == "${no_color}" ]]; then
+    color="$1"; shift
+else
+    if [[ "$2" == "${no_color}" ]]; then
+        color="$2"
+    fi
+fi
 
-remote_urls() {
-  git remote -v
+
+header() {
+    if [[ "${color}" != "${no_color}" ]]; then
+        tput setaf 242
+    fi
+
+    printf "### $1\n"
+
+    if [[ "${color}" != "${no_color}" ]]; then
+        tput sgr0
+    fi
 }
 
-echon() {
-    echo "$@"
-    echo
+git_cmd() {
+    eval "$1"
+    printf "\n"
 }
 
+
 # Show info similar to svn
 
-echo
-echon "## Remote URLs:"
-echon "$(remote_urls)"
+header "Remote URLs:"
+git_cmd "git remote -v"
+
+header "Remote Branches:"
+git_cmd "git branch -r ${color}"
 
-echon "## Remote Branches:"
-echon "$(remote_branches)"
+header "Local Branches:"
+git_cmd "git branch ${color}"
 
-echon "## Local Branches:"
-echon "$(local_branches)"
+if [[ "${max_count}" -ne 1 ]]; then
+    header="Top ${max_count} Commits:"
```

Comment:
IMHO, `Top N Commits:` is vague. Actually we show the last N commits in the current branch.